contribution_to_project.rst

```diff
@@ -0,0 +1,95 @@
+***********************
+Contributing to Ansible
+***********************
+
+This document describes ways how you can contribute to the Open Source projects of the Ansible ecosystem.
+
+.. contents:: Topics
+
+What is a contribution to an Open Source project
+================================================
+
+A contribution to an Open Source project is any good given by an individual or a company to the project including increasing its functionality, improvements of the codebase, code developer experience, user experience, increasing the project's popularity and community, providing infrastructure, and any other kinds of active social or material support.
+
+Most types of contribution to an Open Source project do not require programming skills.
+
+Classification of contributions
+===============================
+
+Contributions can be one of the following types:
+
+  - :ref:`Contribution to a codebase<Contribution to codebase>`.
+  - :ref:`Contribution to code developer experience<Contribution to dev exp>`.
+  - :ref:`Contribution to user experience<Contribution to user exp>`.
+  - :ref:`Contribution to a community<Contribution to community>`.
+  - :ref:`Other kinds of contributions<Other kinds of contributions>`.
+
+.. _Contribution to codebase:
+
+Contribution to a codebase
+--------------------------
+
+Contribution to software itself (its codebase) includes:
+
+  - Proposing new features.
+  - Reporting bugs.
+  - Participating in design and implementation of new components.
+  - Bug fixes.
+  - Code reviewing.
+  - Code refactoring.
+  - Improving stability of an existing codebase through improving a testing codebase and infrastructure.
+  - Maintenance (for example, backporting and releasing).
+
+Examples of codebase contributions which do not require programming skills in scope of the Ansible project can be reviewing proposals, testing pull requests locally, adding integration tests, bug reports, feature requests, providing user experience to programmers in issues and pull requests.
+
+.. _Contribution to dev exp:
+
+Contribution to code developer experience
```

Comment:
Thinking about this, I'd also make all the headers active verbs instead of nouns: `Contributing to the developer experience` and maybe `Finding the right contribution type for you` or something.

Reply:
Sounds good to me, changed, thanks!